queries/wof.jinja2

```diff
@@ -12,7 +12,7 @@ SELECT
       'area', area::bigint,
       'min_zoom', min_zoom,
       'max_zoom', max_zoom,
-      'is_landuse_aoi', CASE WHEN is_landuse_aoi = TRUE THEN TRUE END,
+      'is_landuse_aoi', is_landuse_aoi,
```

Comment:
Yes, that looks good, and similar to what we've got for other tables. Thanks!

The pattern we seem to be following for other tables is to have the `min_zoom` in the `__$LAYER_properties__`. Would we want to do the same here? Or not, given that WOF min/max zoom are intrinsic and not calculated as they are for other layers?